ci-operator/step-registry/osd-ccs/cluster/provision/gcp/osd-ccs-cluster-provision-gcp-commands.sh

```diff
@@ -101,56 +200,39 @@ ocm create cluster ${CLUSTER_NAME} \
 --subscription-type ${SUBSCRIPTION_TYPE} \
 ${MARKETPLACE_GCP_TERMS_SWITCH} \
 ${DISABLE_WORKLOAD_MONITORING_SWITCH} \
-${ETCD_ENCRYPTION_SWITCH}
-"
+${ETCD_ENCRYPTION_SWITCH} --secure-boot-for-shielded-vms"
```

Comment:
Is the option `--secure-boot-for-shielded-vms` required for all of the clusters?

Reply:
Updated, by merging with https://github.com/openshift/release/pull/49942. FYI this option does be required, if using the OSD QE's project `openshift-qe-group-i-osd` which demands enabling the SecureBoot feature.